src/test/java/org/skife/jdbi/v2/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -67,9 +72,10 @@ public Object withHandle(Handle handle) throws Exception
         public String findNameById(@Bind long id);
     }
 
-    @Ignore
     @Test
     public void testFoo() throws Exception {
+        assumeTrue(postgresIsInstalled);
```

Comment:
It looks like a great idea and I've pushed a new version. Thanks for your suggestion!